deployment/hasura/metadata/databases/tables/merlin/mission_model.yaml

```diff
@@ -103,7 +106,7 @@ insert_permissions:
       columns: [mission, name, version, description, jar_id]
```

Comment:
The `default_view` column needs to be added to the `columns` list here and on L113

Reply:
We don't care about this column triggering any of the `event_triggers` right?